docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -210,10 +225,11 @@ In this example we will be deploying an https://github.com/odo-devfiles/nodejs-e
  Experimental mode is enabled, use at your own risk
 
  Validation
- ✓  Checking devfile compatibility [106956ns]
- ✓  Validating devfile component [250318ns]
+  ✓  Checking devfile compatibility [111738ns]
+  ✓  Creating a devfile component from registry: DefaultDevfileRegistry [89567ns]
+  ✓  Validating devfile component [186982ns]
 
- Please use odo push command to create the component with source deployed
+ Please use `odo push` command to create the component with source deployed
```

Comment:
Please remove ` from `odo push` as it messes up the code output :(

Reply:
Haha, markdown formatting is still instinctive for me. I'll fix that